src/components/MapContainer/MapContainer.module.scss

```
@@ -4,7 +4,7 @@
   width: 100%;
   height: 100%;
 
-  .layer-controls {
+  .layer-controls-desktop {
```

With BEM, I believe desktop would be the state. Same with mobile below.
```suggestion
  .layer-controls--desktop {
```